src/devices/Lps22hb/README.md

```diff
@@ -0,0 +1,12 @@
+﻿# LPS22HB - Piezoresistive pressure and thermometer sensor
+
+Some of the applications mentioned by the datasheet:
+
+- Altimeter and barometer for portable devices
+- GPS applications
+- Weather station equipment
+- Sport watches
+
+## References
+
+- <https://www.st.com/resource/en/datasheet/lps22hb.pdf>
```

Comment:
can you add a quick usage in the read me? It' s a great practice even if there is the sample.
Also anything not implemented in this binding? You could list what is and is not implemented.